.gitignore

```diff
@@ -215,4 +215,6 @@ ENV/
 media_volume
 static_volume
 
+poetry.lock
```

Comment:
this should never be in gitignore. this is the whole point of poetry

Reply:
sorry, i found in google gitignore file for poetry

Reply:
done